ide/coqide/coqide.ml

```diff
@@ -1617,13 +1623,13 @@ let build_ui () =
 
   menu help_menu [
     item "Help" ~label:"_Help";
-    item "Browse Coq Manual" ~label:"Browse Coq _Manual"
+    item "Browse Coq Manual" ~label:"Browse Coq _Manual" ~accel:"<Shift>F1" ~stock:`HELP
```

Comment:
You know, this is broken if you use it in a local build of master.  It may work for an officially installed version.  Did you test this at all? (not saying that you have to fix it, though that would great, but adding the accelerator is a bit pointless if the underlying code is broken).

Same thing with Browse Coq Library.  Help for Keyword doesn't seem to do anything useful, either

Reply:
It works with the officially installed version.